ui/analyse/src/study/commentForm.ts

```diff
@@ -135,7 +139,7 @@ export function view(root: AnalyseCtrl): VNode {
               el.onblur = function () {
                 ctrl.focus(false);
               };
-              vnode.data!.hash = current.chapterId + current.path;
+              vnode.data!.path = current.chapterId + current.path;
```

Comment:
I updated this line too since it seemed like a simple typo, even though I think this is not really relavant to any functionality.